libnetwork/netlabel/labels.go

```diff
@@ -59,4 +59,8 @@ const (
 
 	// LocalKVClient constants represents the local kv store client
 	LocalKVClient = DriverPrivatePrefix + "localkv_client"
+
+	// NoProxy6To4 disables proxying from an IPv6 host port to an IPv4-only
+	// container, when the default binding address is 0.0.0.0.
+	NoProxy6To4 = DriverPrivatePrefix + ".no_proxy_6to4"
```

Comment:
I'll update the comment to ...

>	// NoProxy6To4 disables proxying from an IPv6 host port to an IPv4-only
	// container, when the default binding address is 0.0.0.0. This label
	// is intended for internal use, it may be removed in a future release.